src/formio/validators/plugins.js

```diff
@@ -1,41 +1,34 @@
 import {post} from '../../api';
 
-const errorMessageMap = {
-  'kvk-kvkNumber': 'Invalid Kvk Number',
-  'kvk-rsin': 'Invalid RSIN',
-  'kvk-branchNumber': 'Invalid Branch Number',
-  'phonenumber-international': 'Invalid international phonenumber',
-  'phonenumber-nl': 'Invalid Dutch phonenumber',
-};
+export const pluginsAPIValidator = {
+  key: `validate.backendApi`,
+  message(component) {
+    return component.t(component.errorMessage('Invalid'), {
+      field: component.errorLabel,
+      data: component.data,
+    });
+  },
+  check(component, setting, value) {
```

Comment:
In the time validator, we used the pattern:
- `check` returns true/false and sets on the component (under `openForms.validationErrorContext`) a list of error keys.
- `message` processes what is in the `openForms.validationErrorContext` and then decides what error to display.

Not sure if this is good for this case, since the validator returns error messages rather than error messages keys :thinking: